esdt-safe/src/lib.rs

```diff
@@ -252,6 +253,7 @@ pub trait EsdtSafe:
     fn create_transaction_common(
         &self,
         to: EthAddress<Self::Api>,
+        min_bridge_amount: BigUint<Self::Api>,
         opt_refund_info: OptionalValue<RefundInfo<Self::Api>>,
```

Comment:
is this opt_refund_info still used? in which case?

Reply:
Removed